src/hooks/search/utils.ts

```diff
@@ -277,7 +277,7 @@ export const getResultData = (
 ) => {
     return initialSearchData &&
         hasEmptyFilterAndQuery(searchString, allFilters) &&
-        (!sortBy || sortBy === '')
+        (!sortBy || sortBy === '' || sortBy === 'Most Recent')
```

Comment:
Added this fix for another issue DEVHUB-1544. This is actually needed for the pagination to work in topic and topic content pages.